faction.h

```diff
@@ -52,12 +56,10 @@ enum {
 	NATTITUDES
 };
 
-enum {
-	F_WAR,
-	F_TRADE,
-	F_MAGIC,
-	NFACTYPES
-};
+extern const std::string F_WAR;
```

Comment:
if we change FWAR to enum, can we keep the same AString?

Reply:
I would be more afraid of using `AString` instead of `std::string`.

The enum will make code more complicated because it will need those constants anyway (to be used in parsing) and it will need mapping.